engine/script/src/script_sys.cpp

```diff
@@ -1297,6 +1297,45 @@ union SaveLoadBuffer
         return 1;
     }
 
+    //undocummented function for debugger
+
+    static int m_DebuggerLightweightHook;
```

Comment:
We want global variables at the top of the file scope.
We want global variables to start with `g_*`